src/Sarif.Multitool.Library/RewriteOptions.cs

```diff
@@ -16,5 +16,33 @@ public class RewriteOptions : SingleFileOptionsBase
             Default = false,
             HelpText = "Sort results in the final output file.")]
         public bool SortResults { get; set; }
+
+        [Option(
+            "normalize-for-ghas",
```

Comment:
>normalize-for-ghas [](http://example.com/codeflow?start=13&length=18)

don't delete the previous option, mark it as obsolete and make sure it continues to work. update the release notes.

Reply:
fixed